src/ra_server.erl

```diff
@@ -2107,6 +2114,20 @@ make_cluster(Self, Nodes) ->
             Cluster#{Self => new_peer()}
     end.
 
+validate_cluster(State) ->
+    Filter = maps:get(filter_nodes, State),
+    Cluster0 = maps:get(cluster, State),
+    case Filter of
+        undefined ->
+            Cluster0;
+        _ ->
+            maps:filter(fun ({Name, Node}, _) -> Res = lists:member(Node, Filter),
+                                                 Res orelse ?INFO("~p is filtered on node: ~s", [Name, Node]),
```

Comment:
I find this log message to be confusing. "[machine name] is filtered on node […]" does not really tell the user about the consequences.